apps/api/src/app/workflows-v2/usecases/validate-content/build-payload-from-placeholder/build-default-payload.usecase.ts

```diff
@@ -63,6 +63,9 @@ function removeBracketsAndFlattenToNested(input: Record<string, unknown>): Recor
     const cleanedKey = key.replace(/^\{\{|\}\}$/g, '');
 
     const keys = cleanedKey.split('.');
+    if (keys.length === 1) {
+      return;
+    }
```

Comment:
this solves cases if user provides the following control value:
` {{subscriber}} {{subscriber.firstName}}`
in such case API would trow 500 TypeError: Cannot create property 'firstName' on string '{{subscriber}}'